addon/components/country-chooser.js

```diff
@@ -4,14 +4,27 @@ import CountryCodes from '@upfluence/ember-upf-utils/resources/country-codes';
 export default class CountryChooserComponent extends Component {
   codes = CountryCodes
 
+  constructor(owner, args) {
+    super(owner, args);
+
+    if (!args.onCountrySelection) {
+      throw new Error(
+        '[component][country-chooser] Please provide a onCountrySelection action'
+      );
+    }
+  }
+
   // Named Arguments Defaults.
   // -------------------------
   get label() { return this.args.label || 'Country'; }
   get placeholder() { return this.args.placeholder || '-'; }
-  get required() { return this.args.required || false; }
-  get multiple() { return this.args.multiple || true; }
   get dark() { return this.args.dark || false; }
   get size() { return this.args.size || null; }
+  get multiple() {
+    if (this.args.multiple == false) return this.args.multiple;
```

Comment:
this is to avoid `null` value being considering as falsy hence the getter going to the default `true`. We are looking for an explicit `multiple=false`

Reply:
Just a style question. It's ok now to use inline statement ?